psi4/src/psi4/detci/ints.cc

```diff
@@ -376,6 +379,215 @@ void CIWavefunction::setup_mcscf_ints() {
 
     ints_init_ = true;
 }
+void CIWavefunction::setup_mcscf_ints_ao()
+{
+    outfile->Printf("\n   ==> Setting up MCSCF integrals <==\n\n");
+
+    timer_on("CIWAVE: Setup MCSCF INTS AO");
+    if(options_.get_str("SCF_TYPE") == "GTFOCK")
+    {
+ #ifdef HAVE_JK_FACTORY
+        Process::environment.set_legacy_molecule(molecule_);
+        jk_ = boost::shared_ptr<JK>(new GTFockJK(basisset_));
+#else
+        throw PSIEXCEPTION("GTFock was not compiled in this version");
+#endif
+    } else if (options_.get_str("SCF_TYPE") == "DF") {
+        jk_ = JK::build_JK(this->basisset(), get_basisset("DF_BASIS_SCF"), options_);
+    }
+    else if (options_.get_str("SCF_TYPE") == "CD" or options_.get_str("SCF_TYPE") == "PK" or options_.get_str("SCF_TYPE") == "DIRECT")
+    {
+        jk_ = JK::build_JK(this->basisset(), BasisSet::zero_ao_basis_set(), options_);
+    }
+    else {
+        outfile->Printf("\n Please select GTFock, DF, CD or PK for use with MCSCF_TYPE AO");
+        throw PSIEXCEPTION("AO_CASSCF does not work with your SCF_TYPE");
+    }
+    jk_->set_do_J(true);
+    jk_->set_allow_desymmetrization(true);
+    jk_->set_do_K(true);
+    jk_->initialize();
+    jk_->set_memory(Process::environment.get_memory() * 0.8);
+    ints_init_ = true;
+    timer_off("CIWAVE: Setup MCSCF INTS AO");
+}
+void CIWavefunction::transform_mcscf_ints_ao(bool approx_only)
+{
+    //if(approx_only)
```

Comment:
I dont think you have AO rotated builds, it would seem that this needs to be here `if(!approx_only)`.